### DISCUSSION TITLE
I submitted an issue but get deleted

### DISCUSSION THREAD

**Discussion by cryptom-dev** [vercel/next.js#17439] [Help] [answered]
I submitted https://github.com/vercel/next.js/issues/17438 but it gets deleted. I wonder why the issue has been deleted? (I provided the reproduce procedure and my nodejs,next.js,macos version). Maybe security-related issue shouldn't be submitted there? But I am keen on a workaround provided by next.js team :)

**Comment by timneutkens** [comment, marked as answer]
<img width="1138" alt="Screen Shot 2020-09-29 at 16 49 54" src="https://user-images.githubusercontent.com/6324199/94574876-e4ea7f80-0273-11eb-8118-e1fc81abc989.png">

Security related issues should never be reported in the GitHub issue tracker as per our security policy. The issue template selector also shows that (see screenshot above). Instead you should email security@vercel.com if you want to report security-related issues.
